cpp/src/ray/runtime/task/invocation_spec.h

```diff
@@ -0,0 +1,21 @@
+
+#pragma once
+
+#include <ray/core.h>
+#include <msgpack.hpp>
+
+namespace ray {
+namespace api {
+
+class InvocationSpec {
+ public:
+  TaskID taskId;
+  ActorID actorId;
+  int actorCounter;
```

Comment:
I don't see these three fields in the argument list of `TaskExecutor.submitActorTask` in Java. Are they required in C++? Why?

Reply:
I don't ensure want fields required. Maybe we can optimize this in the implementation of cluster mode?